app/views/forms/uploads.html.erb

```diff
@@ -2,16 +2,57 @@
 
 <div class="row">
   <div class="col-md-8 col-sm-7">
-    <h2>Upload Grant Documents</h2>
-    <%= form_for(@upload, multipart: true) do |f| %>
-      <%= hidden_field(:upload, :user_id) %>
-      <%= hidden_field(:upload, :user_type) %>
-      <div class="field">
-        <%= f.file_field :file %>
+    <%= form_for(@upload, html: { class: 'new-upload form-horizontal', multipart: true }) do |f| %>
+      <div class="col-xs-12">
```

Comment:
Now that this isn't inside of a `.row` div, you can actually remove the `col-xs-12` also; it's not doing anything anymore (or you can add a wrapping `.row` around it).  The suggestion I had previously was intended to say that you didn't need to nest a `.form-group` inside of a `.row` as they are effectively the same and set up the row for the bootstrap grid.